iChef-WebUi/src/app/components/recipe-page/recipe-page.component.scss

```diff
@@ -0,0 +1,222 @@
+@import '../../styles/mixins.scss';
+@import '../../styles/variables.scss';
+
+h1 {
+  text-align: center;
+  font-weight: bold;
+
+}
+
+h2 {
+  text-align: center;
+  
+}
+
+
```

Comment:
Same applies here with the blank spaces (see the other comment regarding blank spaces)